DataCleaningScripts/new_moon_numbers.r

```diff
@@ -32,54 +31,66 @@ find_first_trap_night = function(dat) {
 #' @param target_date single date object
 #' @param newmoondates vector of date objects
 #' 
-#' @example closest_newmoon(as.Date('2017-02-26'),moon_dates$NewMOonDate)
+#' @example closest_newmoon(as.Date('2017-02-26'),moon_dates$NewMoonDate)
 #' 
 closest_newmoon = function(target_date,newmoondates) {
   closest = which.min(abs(target_date-newmoondates))
   return(closest)
 }
 
-#' Updates existing version of moon_dates.csv with latest census numbers, rather than recreate the whole file
+#' Updates existing version of moon_dates.csv with latest census numbers
 #'
 #'
 #'
-#' @param final_data_location file path where csv of final data should be saved (in the PortalData repo)
+#' @param
 #' 
-#' @example update_moon_dates(final_data_location='C:/Users/EC/Desktop/git/PortalData/Rodents/moon_dates.csv')
+#' @example update_moon_dates()
 
-update_moon_dates = function(final_data_location) {
+update_moon_dates = function() {
   # load existing moon_dates.csv file
-  moon_dates=read.csv(text=getURL("https://raw.githubusercontent.com/weecology/PortalData/master/Rodents/moon_dates.csv"),stringsAsFactors = F)
+  moon_dates=read.csv("../Rodents/moon_dates.csv",stringsAsFactors = F)
```

Comment:
`FALSE` is considered better than `F`. (Because it is possible to do `F = T` 😱 but not `FALSE = T`)